src/components/sections/ProductSpotlight/ProductSpotlight.tsx

```
@@ -9,11 +9,17 @@ import { Star } from "lucide-react";
 import { Container } from "../../Container";
 
 const ProductSpotlight = () => {
-  const [selectedVariant, setSelectedVariant] = useState(PRODUCT_DATA[0]);
-  const [selectedImage, setSelectedImage] = useState(PRODUCT_DATA[0].src);
-  const [selectedSize, setSelectedSize] = useState(PRODUCT_DATA[0].size[0]);
+  if (!PRODUCT_DATA || PRODUCT_DATA.length === 0) return null;
+  const firstVariant = PRODUCT_DATA[0];
 
-  const thumbnails = selectedVariant.thumbnails;
+  const [selectedVariant, setSelectedVariant] =
+    useState<ProductType>(firstVariant);
```

ESLint

[error] React Hook "useState" is called conditionally. React Hooks must be called in the exact same order in every component render. Did you accidentally call a React Hook after an early return?

(react-hooks/rules-of-hooks)